src/components/slider/slider.scss

```diff
@@ -34,8 +34,11 @@
   font-size: var(--calcite-slider-container-font-size);
 }
 
+// disable browser swipe navigation to next/previous page to prevent
+// interference with the slide thumb intended functionality to follow touch gesture
```

Comment:
Great comment. Can you make it a bit more concise and possibly placed right after `touch-action`?